service/controller/v2/resource/dnsrecord/create.go

```diff
@@ -4,8 +4,9 @@ import (
 	"context"
 
 	providerv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
-	"github.com/giantswarm/azure-operator/service/controller/v2/key"
 	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/azure-operator/service/controller/v2/key"
```

Comment:
The rest is just aligning the imports formatting to maintain the 3 blocks as we have them defined in https://github.com/giantswarm/fmt/blob/master/go/imports.md.